huaweicloud/data_source_huaweicloud_rts_software_config_v1_test.go

```diff
@@ -0,0 +1,65 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+// PASS
```

Comment:
no need to have such unnecessary test result comments "// PASS", please help to clean up all of them.